### ISSUE TITLE
Dimensions .. issue

### ISSUE THREAD

**Issue by khawar85** [peterwittek/somoclu#114]
Dear Respected Friends. 

Good Morning. 
I want to ask a question. After the clustering in SOMOCLU, when I get the get_surface_state to get BMUS.. , i get the array is of like 1523502 and the input was  (951 by 801 = 761751). and the surface should have the values 761751. May I ask where is my mistake and why the values are double. How to resolve the issue? I mean one column is neuron number and other is the BMUS. so, how to get it plotted without neuron number??
my workflow is 

ax = som.get_surface_state()
a = som.get_bmus(ax)
test = np.array[(a)]
em = np.arange(test).reshape((951, 801)))

and error is

  File "<ipython-input-47-fae2bf5ffac3>", line 1, in <module>
    em = np.arange(test).reshape((951, 801))

TypeError: only length-1 arrays can be converted to Python scalars
.. 
This is because the file has neurons and BMUS values. how to get it displayed. if you can guide ..
===================================================
Basically, I want to see the BMUS in the original space back with dimensions. How to see them as map view. This is problem for me. 
++++++++++++++++++++++++++++++++++++++++++++

Please help

Looking forward. 

Waiting for help.

**Comment by khawar85** [comment]
OK.. for now I think I realized what mistake I have done. Thank you for help. if something comes up. I will ask again..

Very nice tool.

**Comment by khawar85** [comment]
At the moment. I have realized my mistake. Thank you..
